11ty/es/definiciones/borrado-bisexual.md

```diff
@@ -0,0 +1,28 @@
+---
+title: borrado bisexual
+slug: borrado-bisexual
+speech: noun
+defined: true
+excerpt: el cuestionamiento o negación de la existencia o legitimidad de la bisexualidad
+reading: 
+  - text: Erasure of Bisexuality
+    href: https://www.glaad.org/bisexual/bierasure
+  - text: 'Gayly: Study finds bi-erasure can lead to depression'
+    href: https://www.gayly.com/study-finds-bi-erasure-can-lead-depression
+---
+
+o borrado bi; el cuestionamiento o negación de la existencia o legitimidad de la [Bisexualidad](/es/definiciones/bisexual);
```

Comment:
In the English version there is an additional phrase that reads 'the shortened form of "bisexual erasure"'. Not sure if this was left out intentionally, but in Spanish I'd say something like "la versión corta de 'borrado bisexual'".